### PR TITLE
Deint

### PR DESCRIPTION
**Pull request by Memphiz**
Added osx and ios overrides and fixed compile of the vtbgles renderer.

@FernetMenta i hope thats ok - at least it worked when i tested it on ios and osx :)

### CODE REVIEW

File: xbmc/cores/VideoPlayer/Process/overrides/ios/ProcessInfoIos.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  CProcessInfo::SetSwDeinterlacingMethods();
+
+  std::list<EINTERLACEMETHOD> methods;
+  {
+    // get the current methods
+    CSingleLock lock(m_videoCodecSection);
+    methods = m_deintMethods;
+  }
+  // add bob deinterlacer for ios
+  methods.push_back(EINTERLACEMETHOD::VS_INTERLACEMETHOD_RENDER_BOB_INVERTED);
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/Process/overrides/ios/ProcessInfoIos.cpp]: I thinks we can drop BOB_INVERTED. you don't want to display the fields in wrong order, do you?

**Comment by FernetMenta** [comment]
looks good, thanks much!
I would drop bob_inverted. I will remove this entirely from the app.

**Comment by Memphiz** [comment]
@FernetMenta removed bob_inverted

**Comment by FernetMenta** [comment]
thanks!

**Pull request merged**